src/components/IssueQueue/IssueQueue.vue

```diff
@@ -36,7 +38,7 @@ const isOpenIssuesSelected = ref(false);
 const tableContainerEl = ref<HTMLDivElement | null>(null);
 const buttonRef = ref<HTMLButtonElement | null>(null);
 const isButtonVisible = useElementVisibility(buttonRef,
-  { rootMargin: '0px 0px 500px 0px', scrollTarget: tableContainerEl },
+  { rootMargin: '0px 0px 500px 0px', scrollTarget: tableContainerEl } as VisibilityOptions,
```

Comment:
Why is this change necessary? Is not raising any errors on my side, have you updated you local dependencies? (Vue and vueuse were updated in #522)

Reply:
I have not, thank you, this solved the error Iw as seeing locally that prompted adding this type assertion.